src/vcGIS.cpp

```diff
@@ -94,3 +94,67 @@ udDouble3 vcGIS_GetWorldLocalUp(const vcGISSpace &space, udDouble3 localCoords)
 
   return udNormalize(upVector - localCoords);
 }
+
+udDouble3 vcGIS_GetWorldLocalNorth(const vcGISSpace &space, udDouble3 localCoords)
+{
+  if (!space.isProjected || space.zone.projection >= udGZPT_TransverseMercator)
+    return udDouble3::create(0, 1, 0);
+
+  udDouble3 northDirection = udDouble3::create(0, 1, 0); //TODO: Fix this
```

Comment:
I'm not sure what there is to fix here? The algorithm seems complete without having to define `northDirection` until 108.